ibmdbpy/frame.py

```diff
@@ -2316,3 +2323,106 @@ def check_numeric_columns(idaobject):
         check_numeric_columns(self)
         if isinstance(other, ibmdbpy.IdaSeries)|isinstance(other, IdaDataFrame):
             check_numeric_columns(other)
+
+###############################################################################
+### New functionalities for IdaDataFrames -- Experimental
+###############################################################################
+    
+    def delete_na(self, columns, logic = "any", inplace = False):
+        """
```

Comment:
The docstring explains the parameters, return values, etc., but doesn't mention what the function actually does.

Reply:
Description added:

> Filter rows containing NULL values. Can be done in a destructive way (rows are deleted in the physical table) or in a non destructive way (a new IdaDataFrame is defined, original table and IdaDataFrame are preserved).